### PR TITLE
Added support for 2019.3 based on the latest EAP release.

### PR DESCRIPTION
**Pull request by kiiadi**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description

Adding required compatibility utils to be able to support 2018.3 through 2019.3

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CompatibilityUtils.kt [hunk 1]
```diff
@@ -23,4 +40,59 @@ object CompatibilityUtils {
...
+            return method.invoke(null, path) as Project
+        }
+        val legacyCreate = PlatformTestCase::class.java.getMethod("createProject", File::class.java, String::class.java)
+        return legacyCreate.invoke(null, path.toFile(), "Fake") as Project
+    }
+
+    /**
+     * FIX_WHEN_MIN_IS_192 WebModuleType.getInstance() is deprecated in 191, use WebModuleTypeBase#getInstance for version 191+
+     */
+    @TestOnly
```
Comment (hunterwerlla) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CompatibilityUtils.kt]: This is called from `NodeJsRuntimeGroup.kt` so it's definitely not `@TestOnly`

File: jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CompatibilityUtils.kt [hunk 2]
```diff
@@ -23,4 +40,59 @@ object CompatibilityUtils {
     @JvmStatic
     fun createPopupBuilder(validationInfo: ValidationInfo, configurator: Consumer<JEditorPane>?): ComponentPopupBuilder? =
         ComponentValidator::class.staticFunctions.find { it.name == "createPopupBuilder" }?.call(validationInfo, configurator) as? ComponentPopupBuilder
+
+    /**
+     * Can be removed when min-version is 19.1 FIX_WHEN_MIN_IS_191
```
Comment (hunterwerlla) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CompatibilityUtils.kt]: All things that are actually `FIX_WHEN_MIN_IS_191` are currently `FIX_WHEN_MIN_IS_192` to have a single string to look for

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CompatibilityUtils.kt]: ? the idea here would be that there's a common pattern string to look for when we move off a version ... not sure what you mean

File: jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CompatibilityUtils.kt (hunk 1 above)
Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/utils/CompatibilityUtils.kt]: yes - good catch

**Pull request merged**